tests/aws/services/cloudformation/resources/test_lambda.py

```diff
@@ -6,6 +6,7 @@
 import pytest
 from localstack_snapshot.snapshots.transformer import SortingTransformer
 
+from aws.services.lambda_.event_source_mapping.utils import is_v2_esm
```

Comment:
```suggestion
from tests.aws.services.lambda_.event_source_mapping.utils import is_v2_esm
```

Reply:
thanks (PyCharm auto-import after stashing away the other fixes) 👍